emuu-client/src/App.js

```diff
@@ -1,16 +1,14 @@
 import logo from "./logo.svg";
 import "./App.css";
 import { createTheme, ThemeProvider } from "@mui/material/styles";
-import HeaderPostLogin from './components/NavbarPostLogin/HeaderPostLogin'
-import {BrowserRouter, Route, useHistory} from 'react-router-dom'
-import Login from './components/UserAuthentication/LoginScreen'
```

Comment:
Delete both LoginScreen and RegisterScreen as imports since they are no longer in use